specification/indices/stats/types.ts

```diff
@@ -85,6 +91,10 @@ export class IndicesStats {
   shards?: Dictionary<string, ShardStats[]>
   total?: IndexStats
   uuid?: Uuid
+  /** @since 8.1.0 */
+  health?: HealthStatus
+  /** @since 8.1.0 */
+  status?: string
```

Comment:
I was surprised this wasn't an enum yet! Should we add a new enum `IndexState` with `open` and `closed` to mirror the enum in Elasticsearch?

Reply:
Same here 😄  I was a bit cautious to add an enum directly since this is a new field/value - I'll check the server code and adjust to enum.

Reply:
https://github.com/elastic/elasticsearch/blob/master/server/src/main/java/org/elasticsearch/cluster/metadata/IndexMetadata.java#L125 

`IndexMetadata.State` is an enum `open` & `close`